docs/rules/no-async-actions.md

```diff
@@ -0,0 +1,53 @@
+# Do not use async actions
+## Rule `no-async-actions`
```

Comment:
Instead, could we throw a lint error _only_ after an `await` AND access of `this`?

Reply:
example?: https://github.com/ember-cli/eslint-plugin-ember/pull/1421